web_dashboard_tile/README.rst

```diff
@@ -36,6 +41,8 @@ Roadmap
 * Add icons.
 * Support client side action (like inbox).
 * Restore original Domain + Filter when an action is set.
```

Comment:
There's usage help on the field: `Python Format String valid with str.format()`
I think it's a good idea to add some examples there, though.